src/test/ui/as-precedence.rs

```diff
@@ -1,5 +1,7 @@
 // run-pass
 
+#[allow(unused_parens)]
```

Comment:
Either make this `#![allow(unused_parens)]` or remove the blank line underneath it, so it's clear it's referring specifically to `main`.